rpcs3/Emu/Cell/lv2/sys_game.cpp

```diff
@@ -1,16 +1,74 @@
 #include "stdafx.h"
 #include "util/sysinfo.hpp"
+#include "util/v128.hpp"
 #include "Emu/Memory/vm_ptr.h"
 #include "Emu/Cell/ErrorCodes.h"
 #include "Emu/System.h"
-
+#include "Emu/system_utils.hpp"
 #include "Emu/IdManager.h"
 #include "Utilities/Thread.h"
 
 #include "sys_game.h"
 
 LOG_CHANNEL(sys_game);
 
+struct board_storage
+{
+public:
+	bool read(void* buffer)
+	{
+		if (!buffer)
+			return false;
+
+		const auto data = storage.load();
+		memcpy(buffer, &data, size);
+
+		return true;
+	}
+
+	bool write(void* buffer)
+	{
+		if (!buffer)
+			return false;
+
+		storage.store(*(be_t<v128>*)buffer);
```

Comment:
This is undefined behavior (breaks 'type aliasing' rule). You need to use memcpy or read_from_ptr.

Reply:
Thanks for the notice.
I've fixed it accordingly.